### DISCUSSION TITLE
heating circuit 3

### DISCUSSION THREAD

**Discussion by colinfrei** [pail23/stiebel_eltron_isg_component#120] [Q&A]
Hi,

I have a third heating circuit, which I see in ISG. Afaict this isn't around in Home Assistant.
Am I correct in assuming that this is not made available via modbus? I only see HK 1 and HK 2 listed here: https://www.stiebel-eltron.ch/content/dam/ste/ch/de/downloads/kundenservice/smart-home/Modbus/Modbus%20Bedienungsanleitung.pdf

Thanks for your help :)

**Comment by pail23** [comment]
This should be implemented in the meanwhile. Please check the latest version 2024.1 and let me know if you find the correct values

**Comment by tomnold** [comment]
I have an issue with HK 3 as well. In the ISG Dashboard it shows me the correct Target Temp for HK3, in HA the Value for the Target Temp ist actually the actual Temp. Can I somewhere change that (I was not able up to now...)

**Reply by LaurinD** [reply]
i coded the HK3. its already in the aktual version

**Comment by Optic00** [comment]
Heating circuit 3 is supported. The incorrect target temperature reported later in this thread was a separate register shift, documented and fixed in issue #560 and included since release 2026.8. Closing this discussion. If HK3 still shows incorrect values, please open a new issue with the controller and integration versions.
